packages/core/src/WebPlayer.ts

```diff
@@ -143,6 +143,35 @@ export default class WebPlayer extends EventEmitter {
     }
   }
 
+  changeVolume(change) {
+    if (change > 0 && this.tech.volume < 1) {
```

Comment:
the code here does the exact same thing, could just be in a single if statement.